### DISCUSSION TITLE
After i log out still i can browse and download anything i want

### DISCUSSION THREAD

**Discussion by Officepoint** [rejetto/hfs#435] [General]
Hi!
Why, after logging out, can I still browse and download whatever I want, even though I am no longer logged in. I have tried with every account created and the problem persists with all of them.
Thanks a lot!

**Comment by rejetto** [comment]
please provide more information.
You can browse and download by default. Did you set permissions to prevent that?

**Comment by Officepoint** [comment]
Hi Rejjeto!
When I access 192.168.18.88, this is my ip adress which i setup, I am
automatically directed to the folder without logging in. If I log in with
any other account I can view/download the folder, but after I log out I can
still browse and download anything I want.
I'm thinking that after I log out I should not be able to see any folder
and if I try again to access the ip address it will ask for my credentials.
I don't know exactly what the permissions are to prevent this, maybe I did
something wrong but I can't figure it out.
Thank you !

**Comment by rejetto** [comment]
to prevent people from seeing stuff without being logged in, go to Admin-panel > shared files > select the folder > set the "who can download" permission

**Comment by Officepoint** [comment]
Thanks Rejetto! It worked!
So basically every folder must have permissions, Who can download...,
otherwise it is seen by everyone if it is left with Anyone permission.
If I set a password for the Admin account and I forgot it, how can I reset
it?

Thanks!

**Comment by rejetto** [comment]
you can also set permissions only once on the home/root, and sub-folders will inherit such permission.

if you lose access to admin, the easiest thing to do is to go on the machine where hfs is running and use a browser on localhost. By default you enter without a password.

otherwise you can change the password of any account by editing file config.yaml with a text editor, see these instructions
https://github.com/rejetto/hfs/blob/main/config.md#accounts

**Comment by Officepoint** [comment]
Thank you very much Rejetto! I wil try!
Is there any solution to connect remotely as localhost to set permissions
on folders?
What I mean is in case if I am not at home to work directly on the pc, how
can I access localhost?
I don't mean to log in remotely with Andesk TeamViewer, etc, on my home pc.
If I can access the fileserver from anywhere, for example, at
http://cars.com is there any option to see the localhost at say
http://cars.com/localhost or something.
I don't know if I explained the best, unfortunately English is not my
native language.

Thanks!!

**Comment by rejetto** [comment]
you are missing the point: with hfs i decided to give this special role to localhost exactly because people CANNOT access it, so you must be the owner.
So, if you don't have localhost, you need an admin account.
what's your language? i'm italian

**Comment by Officepoint** [comment]
Thank you Rejjeto!
Yesss, I discovered now, i have to press the Options button, Options>Admin-panel
All the best!!
Mihai!
